templates/openwisp2/settings.py

```diff
@@ -494,3 +506,29 @@
 {{ instruction }}
 
 {% endfor %}
+
+{% if django_cors.enabled %}
+
```

Comment:
to avoid scattering too many ifs in this file, what about concentrating all the instructions for cors here?

Eg:

- place de imports here
- `INSTALLED_APPS.append('corsheaders')`
- `MIDDLEWARES.append(...)`

Reply:
Yes, It would be better to place all settings related to `cors` at one place 👍